### PR TITLE
Add get_scale_{seconds,fs}() to parse SDF

### PR DESCRIPTION
**Pull request by elmsfu**
`get_scale_seonds()` returns timescale in seconds
`get_scale_fs()` return timescale as int femtoseconds

### CODE REVIEW

File: sdf_timing/utils.py [hunk 1]
```diff
@@ -1,3 +1,42 @@
+import re
+
+
+def get_scale(timescale):
```
Comment (mithro) [inline sdf_timing/utils.py]: I think we might want a `def get_scale_ps(timescale) -> int:` (IE a version which returns int / long values).

File: sdf_timing/utils.py [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+    mm = re.match(r'(10{0,2})(\.0)? *([munpf]?s)', timescale)
+    sc_lut = {
+        's': 1.0,
+        'ms': 1e-3,
+        'us': 1e-6,
+        'ns': 1e-9,
+        'ps': 1e-12,
+        'fs': 1e-15,
+    }
+    ret = None
```
Comment (mithro) [inline sdf_timing/utils.py]: If there is no match, you get none back? Is that the right thing to do? Or should it raise an error?

File: sdf_timing/utils.py [hunk 3]
```diff
@@ -1,3 +1,42 @@
...
+    >>> get_scale('10 ns')
+    1e-08
+
+    >>> get_scale('10.0 us')
+    9.999999999999999e-06
+
+    >>> get_scale('100.0ms')
+    0.1
+
+    >>> get_scale('100 s')
```
Comment (mithro) [inline sdf_timing/utils.py]: Can the scale string be in other floating point forms? (IE scientific?)

Comment (elmsfu) [reply sdf_timing/utils.py]: No the format is pretty strict in the SDF spec

File: sdf_timing/utils.py (hunk 1 above)
Comment (elmsfu) [reply sdf_timing/utils.py]: As soon as I created the PR I remembered I didn't like the example that gives 9.9....  I'll add the ps variant.

File: sdf_timing/utils.py (hunk 2 above)
Comment (elmsfu) [reply sdf_timing/utils.py]: Probably a good idea. I'll change to follow errors in the rest of the package.

Comment (elmsfu) [reply sdf_timing/utils.py]: Done

File: tox.ini [hunk 4]
```diff
@@ -27,6 +27,7 @@ commands =
     # used by parser. This triggers flake8 error. Let's ignore this for now.
     flake8 --ignore F401 .
     py.test tests
+    py.test --doctest-modules sdf_timing
```
Comment (mithro) [inline tox.ini]: I think you can add a flag to py.test to find doctests?

Comment (elmsfu) [reply tox.ini]: I think that's what I've done with `--doctest-modules`. I may be able to combine the two lines and run at the top level. Is that what you mean?

Comment (mithro) [reply tox.ini]: I was thinking it was part of https://docs.pytest.org/en/latest/goodpractices.html

Maybe it is just `py.test --doctest-modules` without any arguments?

Comment (mithro) [reply tox.ini]: FYI I would just log a github issue to investigate and merge this as is...

**Pull request merged**